### PR TITLE
Re-enable PPC tests

### PR DESCRIPTION
**Pull request by h-vetinari**
Now that travis is back, let's try to run the PPC tests; based on #263 to avoid more conflicts in `conda-forge.yml`

### CODE REVIEW

**Comment by h-vetinari** [comment]
I don't understand what's up with Travis. All the logs here are cut off extremely early in the build (also in raw log), e.g.
```
+docker pull quay.io/condaforge/linux-anvil-ppc64le
Using default tag: latest
latest: Pulling from condaforge/linux-anvil-ppc64le
3fe478aaff9b: Pulling fs layer
ff29e75e0dc0: Pulling fs layer
3fe478aaff9b: Verifying Checksum
3fe478aaff9b: Download complete
3fe478aaff9b: Pull complete
```

**Comment by h-vetinari** [comment]
Still weird things happening on travis, like the log cutting out and then reappearing in the middle of the test suite:
```
+ mamba update --update-specs --yes --quiet --channel conda-forge conda-build pip boa conda-forge-ci-setup=3
........................................................................ [ 13%]
........................................................................ [ 13%]
.s........................s..x.......................................... [ 14%]
```
or
```
Attempting to finalize metadata for numpy
conda-forge/linux-ppc64le                                   Using cache
conda-forge/noarch                                          Using cache
Reloading output folder: /home/conda/feedstock_root/build_artifacts
.......................................................................... [ 73%]
........................................................................ [ 74%]
........................................................................ [ 74%]
```

Reloading the same job again removes all the intervening progress bars that had been printed.

**Comment by h-vetinari** [comment]
@isuruf

Good news, this seems to be passing on PPC! 🥳 

However, some jobs have garbled/non-existent logs - are we comfortable with trusting that the uploads will work correctly even if we cannot introspect the logs?

**Comment by h-vetinari** [comment]
@isuruf, any comment on your confidence in Travis, given that the logs do not seem introspectable?

It'd be nice to run the test suite for PPC again.

**Comment by h-vetinari** [comment]
Ping @isuruf

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,6 +49,10 @@ requirements:
     - numpy-base <0a0
 
 {% set tests_to_skip = "_not_a_real_test" %}
+# https://github.com/numpy/numpy/issues/15243
+{% set tests_to_skip = tests_to_skip + " or test_loss_of_precision[complex256]" %}  # [ppc64le]
+# Probably needs too much memory on ppc64le, it sometimes crashes on travis
+{% set tests_to_skip = tests_to_skip + " or test_large_archive" %}                  # [ppc64le]
```
Comment (isuruf) [inline recipe/meta.yaml:55]:
```suggestion
{% set tests_to_skip = tests_to_skip + " or test_large_archive" %}                  # [CI == "travis"]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -59,7 +63,7 @@ test:
   requires:
     - pytest
     - pytest-timeout
-    - pytest-xdist
+    - pytest-xdist         # [not ppc64le]
```
Comment (isuruf) [inline recipe/meta.yaml:66]:
```suggestion
    - pytest-xdist         # [CI != "travis"]
```

**Review by isuruf — APPROVED** [review]
Minor change needed

**Comment by h-vetinari** [comment]
Thanks for taking a look. I don't mind the changes, but IMO they're redundant in the sense that we cannot run ppc-tests on azure either (due to the emulation problems; wondering if qemu 6.2 or 7.0 would help, but https://github.com/multiarch/qemu-user-static hasn't updated yet).

**Pull request merged**